infrastructure/src/test/java/org/corfudb/infrastructure/management/NetworkStretcherTest.java

```diff
@@ -0,0 +1,58 @@
+package org.corfudb.infrastructure.management;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+import java.time.Duration;
+
+public class NetworkStretcherTest {
```

Comment:
There is no test testing the decrement of the `currentPeriod`.

Reply:
good point, thanks, will add

Reply:
Done